planner/planner_APIs.go

```diff
@@ -95,6 +97,7 @@ type PlaceDetailsResp struct {
 	PhotoURL        string
 }
 
+// redundant?
 type TripShowDetails struct {
 	City    string `form:"city"`
 	Country string `form:"country"`
```

Comment:
This seems to be redundant. Consider removing it.

Reply:
Done